src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHet.java

```diff
@@ -58,8 +58,9 @@
         final double pval = exactTest(hetCount, refCount, homCount);
 
         //If the actual phredPval would be infinity we will probably still filter out just a very large number
+        //Since the method does not guarantee precision for any p-value smaller than 1e-16, we can return the phred scaled version
         if (Math.abs(pval - 0.0) < 10e-60) {
```

Comment:
I think this is the same as (pval < 10e-60) unless the absolute value was to protect against some kind of overflow that I'm not seeing? I changed it to pval < 10e-60.